src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
You use this URL in a lot of places. Create file api/constants.ts, move this variable there, and use it from there